fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -194,6 +194,7 @@ def predict(self, X):
         :rtype: Scalar or vector
         """
         positive_probs = self._pmf_predict(X)[:, 1]
+        print(positive_probs)
```

Comment:
you probably meant to remove those print statements here.

Reply:
Yes! Thank you